packages/core/src/components/StepList/Step.jsx

```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import StepLink from './StepLink';
+import SubStep from './SubStep';
+import classNames from 'classnames';
+import { stepShape } from './StepList';
+
+export const Step = ({ step, ...props }) => {
+  const start = step.isNextStep;
+  const resume = step.started && !step.completed;
+  const className = classNames('ds-c-step', {
+    'ds-c-step--current': start || resume,
+    'ds-c-step--completed': step.completed
+  });
+  const contentClassName = classNames('ds-c-step__content', {
+    'ds-c-step__content--with-content': step.description || step.steps
+  });
+  // const actionsId = `step-${step.id}-actions`;
+  // const subActionsId = `step-${step.id}-sub-actions`;
+  return (
+    <li className={className}>
+      <div className={contentClassName}>
+        <h2 className="ds-c-step__title">{step.title}</h2>
```

Comment:
@pwolfert In our design system sync today, @cindyphan @1Copenut and I chatted and decided that from an accessibility and usability standpoint, we'd like to also link this title to the same place as the primary Start/Resume/Edit button, when the step is completed or in progress. That would also mean that this title would be the default link style (blue and underlined) for both of these states.

Reply:
I'll upload the change soon. The titles are so ugly now, but I guess that's the price we pay for usability.

Reply:
I know it's not as consistent, but the link for completed titles might almost be bearable to my eyes if it were black.

Reply:
Hm..I don't find it to be that ugly, but this is probably one of those subjective things and influenced by how you've become accustomed to seeing it. I think the styling we use for links should be as consistent as possible. Here's how it looked in the original design comps for App 3:

![image](https://user-images.githubusercontent.com/371943/33622450-d9adca52-d9bb-11e7-8653-94333c1ad278.png)